src/lumigo_tracer/parsers/utils.py

```diff
@@ -289,9 +290,25 @@ def _parse_streams(event: dict) -> Dict[str, str]:
         result["messageId"] = event["Records"][0].get("messageId")
     elif triggered_by == "kinesis":
         result["messageId"] = safe_get(event, ["Records", 0, "kinesis", "sequenceNumber"])
+    elif triggered_by == "dynamodb":
+        result.update(_parse_dynamomdb_event(event))
     return result
 
 
+def _parse_dynamomdb_event(event):
+    creation_time = (
+        event["Records"][0].get("dynamodb", {}).get("ApproximateCreationDateTime", 0) * 1000
```

Comment:
Why ` * 1000`? Maybe extract it to a function so it will be more clear.

Reply:
seconds to milliseconds..
I will 👍